### DISCUSSION TITLE
I have tired to resubmitted my application.

### DISCUSSION THREAD

**Discussion by dhroshan** [community/community#51042] [GitHub Education]
![Screenshot 2023-03-26 102553](https://user-images.githubusercontent.com/38531559/227756286-859bff7f-eba1-4ed4-b600-3fb07eb1e388.png)
![Screenshot 2023-03-26 102626](https://user-images.githubusercontent.com/38531559/227756293-a703fa3b-f522-4c61-bc5d-8b02f110f794.png)
First, I was told to put the information correctly.  I made it and resubmitted my application.  Now, this error is showing.  Can I fix it?

**Comment by samus-aran** [comment]
Hi @dhroshan 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
